assets/src/blocks/ActionsList/index.js

```diff
@@ -21,7 +21,7 @@ export const ACTIONS_LIST_BLOCK_ATTRIBUTES = {
   className: 'actions-list p4-query-loop is-custom-layout-grid',
   query: {
     pages: 0,
-    perPage: 3,
+    perPage: 100,
```

Comment:
@comzeradd in terms of performance (and also maybe design) should make this limit lower?